configs/peer/config_test.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
If you need this file, it needs to be placed elsewhere.

Reply:
Given that this is used for testing of the `config` crate, I'd suggest putting it there locally.